spec/patterns/httpd_spec.rb

```diff
@@ -2,6 +2,18 @@
 require "spec_helper"
 require "logstash/patterns/core"
 
+describe "COMBINEDAPACHELOG" do
+
+  it "matches an email address in the ident field" do
+    expect(subject).to match("1.2.3.4 an.email@address.com - [22/Sep/2015:14:39:21 +0000] \"GET /index.html\" 200 182 \"https://my.website.com/index.html\" \"Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2228.0 Safari/537.36\"")
+  end
+
+  it "matches an email address in the auth field" do
+    expect(subject).to match("1.2.3.4 - an.email@address.com [22/Sep/2015:14:39:21 +0000] \"GET /index.html\" 200 182 \"https://my.website.com/index.html\" \"Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2228.0 Safari/537.36\"")
+  end
+
+end
```

Comment:
good additions to the test suites, I wonder if you could also add some that match the "former" USER pattern too? like this we'll be sure there are no regressions introduced.